pkg/update/update.go

```diff
@@ -29,7 +31,21 @@ func InitAvailableUpdatesDir() error {
 }
 
 func GetAvailableUpdates(kotsStore storepkg.Store, app *apptypes.App, license *kotsv1beta1.License) ([]types.AvailableUpdate, error) {
-	updateCursor, err := kotsStore.GetCurrentUpdateCursor(app.ID, license.Spec.ChannelID)
+	var err error
+	var licenseChan *kotsv1beta1.Channel
+	if app.ChannelID == "" {
```

Comment:
would using `GetOrBackfillLicenseChannel` suffice here?

Reply:
Yep i can switch over to using that. On these we already know app.ChannelID is empty, so calling GetOrBackfill generates an extra call to the db to see if its present. That's really the only reason these where backfilling directly instead.

Reply:
> On these we already know app.ChannelID is empty

how so? wouldn't it be possible that it go backfilled at some point?

Reply:
also, the extra db call would only happen once to backfill the channel id, no?

Reply:
The `GetOrBackfillLicenseChannel` method queries the app table for the channel id using the app id provided. Then either just returns that channel id if present or backfill's if we didn't have one (and then returns the backfilled id). 

But in some of these call locations we've just recently called `store.GetApp()`. So `app.ChannelID` was already fresh. On those instead of calling `GetOrBackfillLicenseChannel` i was just backfilling to skip having to hit the table again just to find out that channel id is empty.